util/grpc/tests/certs/server1.crt

```diff
@@ -0,0 +1,31 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Is this something where we want to generate these rather than commit tot he repo? It looks like the generate.sh script is generating them?

Reply:
I don't see a reason to do that, they're good for ten years and it simplifies running the tests.

Reply:
Every private key in the repo is an exception we have to carry around on an spreadsheet listing exceptions to audit results, and audit results demonstrating the exceptions---i.e. "these are our audits, and we ignored result 5.2.1(b) because it's a test key, and we promise we've never used it it in production, which you can see from this other audit result of our operations which guarantees that"---and discuss on calls with more staid institutions. Then some enterprising hack on the infosec review call asks "but you're a distributed system, can you guarantee nobody is using this compromised-in-advance key before I grant approval for this integration to continue"? 

If the key is never in the repo, that conversation never has to happen.